src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/events/PluginPatchsetCreatedEvent.java

```diff
@@ -109,6 +112,10 @@ public boolean shouldTriggerOn(GerritTriggeredEvent event) {
         if (!super.shouldTriggerOn(event)) {
             return false;
         }
+        // if it was a manually triggered event, always honor it
```

Comment:
Of course it's debatable.
My idea was that when you explicitely trigger something, you definitely expect something to happen.
We don't exclude drafts in our settings, and I would suspect that doing so means that you want to have very early drafts that often don't compile. But in this use-case, you probably still want to be able to verify a draft manually.
OTOH, if there is no job watching the corresponding project, you still won't have any build launched...
It may make sense to have this behavior configurable, but then should it be a checkbox such as "Latest Patchset Only" on the manual trigger screen, or a part of the job configuration, or both (and then which one takes precedence) ? I went for simple and (to my eyes at least) reasonable enough, since I really wanted to be able to edit commit messages at (almost) no cost : some of our devs have *a lot* of progress margin there !